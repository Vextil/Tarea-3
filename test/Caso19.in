# stress Sistema
# Prueba de uso de memoria
MemoriaSistema
Fin
