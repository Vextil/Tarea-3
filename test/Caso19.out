1># stress Sistema
2># Prueba de uso de memoria
3>Fin Apagar.Fin Solicitan.
4>Fin
